extensions/amp-analytics/0.1/events.js

```diff
@@ -1099,6 +1104,40 @@ export class VideoEventTracker extends EventTracker {
         }
       }
 
+      if (normalizedType === VideoAnalyticsEvents.PERCENTAGE_PLAYED) {
+        if (!percentages) {
+          user().error(TAG,
+              'video-percentage-played requires percentages spec.');
+          return;
+        }
+
+        if (!percentages.every(
+            percentage =>
+              percentage > 0 && (percentage % percentageInterval) == 0)) {
+
+          user().error(TAG,
+              'Percentages must be set in increments of %s with non-zero ' +
+                'values',
+              percentageInterval);
```

Comment:
I think we need to return early in error case here but we are in a `every` so either switch to for..loop or set a flag and return based on that.